src/whylogs/features/transforms.py

```diff
@@ -0,0 +1,125 @@
+
+import numpy as np
+from typing import Union, Callable, List
+from PIL.Image import Image as ImageType
```

Comment:
FYI direct import from PIL will prevent users from accessing classes here

Reply:
These are all image based, as we move to tensors we refactor it further.